src/common/transformations/src/transformations/common_optimizations/fuse_rotary_positional_embeddings.cpp

```diff
@@ -854,6 +856,80 @@ ov::pass::RoPEFusionChatGLM::RoPEFusionChatGLM(int split_output_id, const bool s
     this->register_matcher(m, callback);
 }
 
+ov::pass::RoPEFusionChatGLMHF::RoPEFusionChatGLMHF() {
+    MATCHER_SCOPE(RoPEFusionChatGLMHF);
```

Comment:
Could you please check if this code fix is working well on the CPU for the issued model?